Description: uv backported to system ev
 Upstream changed its own copy of libev to gain some performance.
 This patch favors maintainability by restoring libev compatibility.
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: not-needed
Last-Update: 2012-10-05
--- a/deps/uv/src/unix/core.c
+++ b/deps/uv/src/unix/core.c
@@ -242,9 +242,8 @@
 
 
 static void uv__poll(uv_loop_t* loop) {
-  void ev__run(EV_P_ ev_tstamp waittime);
   ev_invoke_pending(loop->ev);
-  ev__run(loop->ev, uv__poll_timeout(loop) / 1000.);
+  ev_run(loop->ev, EVRUN_NOWAIT);
   ev_invoke_pending(loop->ev);
 }
 
